### PR TITLE
test: use assert.strictEqual()

### PR DESCRIPTION
**Pull request by Nishikoh**
Fix https://github.com/nodejs/node-addon-api/issues/775

### CODE REVIEW

File: test/threadsafe_function/threadsafe_function_existing_tsfn.js [hunk 1]
```diff
@@ -12,8 +12,8 @@ module.exports = Promise.all([
...
   const testCall = binding.threadsafe_function_existing_tsfn.testCall;
   
-  assert(typeof await testCall({ blocking: true,  data: true  }) === "number");
-  assert(typeof await testCall({ blocking: true,  data: false }) === "undefined");
-  assert(typeof await testCall({ blocking: false, data: true  }) === "number");
-  assert(typeof await testCall({ blocking: false, data: false }) === "undefined");
+  assert.strictEqual(typeof await testCall({ blocking: true,  data: true  }) , "number");
+  assert.strictEqual(typeof await testCall({ blocking: true,  data: false }) , "undefined");
+  assert.strictEqual(typeof await testCall({ blocking: false, data: true  }) , "number");
+  assert.strictEqual(typeof await testCall({ blocking: false, data: false }) , "undefined");
```
Comment (NickNaso) [inline test/threadsafe_function/threadsafe_function_existing_tsfn.js:18]: @Nishikoh could you remove the space before the comma ","?
Es.
```js
assert.strictEqual(typeof await testCall({ blocking: false, data: false }), "undefined");
```

Comment (Nishikoh) [reply test/threadsafe_function/threadsafe_function_existing_tsfn.js]: Thank you for the review. Fixed it.

**Comment by gabrielschulhof** [comment]
CI:
| Version | Job | Status |
|---:|---|:---:|
| v15.x | https://ci.nodejs.org/job/node-test-node-addon-api-new/2526/ |  |
| v14.x | https://ci.nodejs.org/job/node-test-node-addon-api-new/2527/ |  |
| v12.x | https://ci.nodejs.org/job/node-test-node-addon-api-new/2528/ |  |
| v10.x | https://ci.nodejs.org/job/node-test-node-addon-api-new/2529/ |  |

**Comment by gabrielschulhof** [comment]
On v15.x it fails for unrelated reasons – specifically, because the `SetImmediate()`ed finalizers have landed in core. Thus, getting the tests to pass may depend on https://github.com/nodejs/node-addon-api/pull/773.

**Comment by gabrielschulhof** [comment]
Rebased.

**Comment by gabrielschulhof** [comment]
CI:

| Version | Job | Status |
|--:|---|:---:|
| v15.x | https://ci.nodejs.org/job/node-test-node-addon-api-new/2552/ |  |
| v14.x | https://ci.nodejs.org/job/node-test-node-addon-api-new/2553/ |  |
| v12.x | https://ci.nodejs.org/job/node-test-node-addon-api-new/2554/ |  |
| v10.x | https://ci.nodejs.org/job/node-test-node-addon-api-new/2555/ |  |

**Comment by gabrielschulhof** [comment]
Landed in 4ce40d22a62d2c784b009179b74bfa1d0b5c810d.

**Pull request closed without merging**